glom/core.py

```diff
@@ -145,8 +187,9 @@ def __repr__(self):
         return '%s(%r, %r, %r)' % (cn, self.exc, self.path, self.part_idx)
 
     def __str__(self):
-        return ('could not access %r, part %r of %r, got error: %r'
-                % (self.path.values()[self.part_idx], self.part_idx, self.path, self.exc))
+        return GlomError.__str__(self) + (
+            'could not access %r, part %r of %r, got error: %r'
+            % (self.path.values()[self.part_idx], self.part_idx, self.path, self.exc))
```

Comment:
I guess super() might be a bit vague with this much multiple inheritance, is that right?